src/assets/content/dictionary/java.md

```diff
@@ -1,6 +1,15 @@
 ---
-title: Java
+title: Application Programming Interface (API)
 definition: ''
 perspectives: []
 
```

Comment:
```suggestion
```
Bug: The repeated keys here break the build, so you can remove them and it should unbreak the build.